src/Components/TodoItems.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
Remove file to `TodoItem.tsx` 
<img width="371" alt="image" src="https://github.com/user-attachments/assets/e13bd6d9-f356-4a50-bd05-88db0194f000" />